ts_src/psetv2/zkp.ts

```diff
@@ -284,7 +285,8 @@ export class ZKPGenerator {
 
       if (input.issuanceInflationKeys! > 0) {
         const token = input.issuanceInflationKeys!.toString(10);
-        const asset = input.getIssuanceInflationKeysHash();
+        const entropy = input.getIssuanceEntropy();
+        const asset = calculateReissuanceToken(entropy, true);
```

Comment:
why hardcoding `true` when we can infer this info from `input.blindedIssuance`?

Reply:
@louisinger by looking at this `blindIssuances()` function, I don't see any check on `input.blindedIssuance` to make sure any blinding is required.
I can now understand why the flag is hardcoded to `true`, but in general we should have a variable containing the value depending on `input.blindedIssuance` used to know if we need to produce any commitment/proof for the current input issuance.